aya/src/maps/lpm_trie.rs

```diff
@@ -63,17 +69,21 @@ pub struct LpmTrie<T, K, V> {
 /// let ipaddr = Ipv4Addr::new(8,8,8,8);
 /// let key =  Key::new(16, u32::from(ipaddr).to_be());
 /// ```
+
 #[repr(packed)]
 pub struct Key<K: Pod> {
-    /// Represents the number of bytes matched against.
```

Comment:
Why did you remove those docstrings?

Reply:
Because the struct members are not `pub` anymore.
The docstrings re-appear on the public accessors https://github.com/aya-rs/aya/pull/526/files/9c7f415a1b8217b95b803d288feb8ce6daff27dc#diff-f6d6661573434c62eecc651f4986dfa4e5235ad59ee99bcb4478a370a7fb8621R100

Reply:
Ah, true